src/CRISP.BackendChallenge/Repository/ContextRepository.cs

```diff
@@ -20,36 +21,97 @@ public IQueryable<T> Query()
     /// <inheritdoc />
     public IEnumerable<T> GetAll()
     {
-        throw new NotImplementedException();
+        var entities = _context.Employees.Include(e => e.Logins).ToList();
```

Comment:
why keep these methods or even the repository itself generic? This method will **never** support any types other than `Employee` for `T`.

Do you see value in returning `null` when this method is called with a type parameter other than `Employee`?